src/components/shared/thread/ActivityGraph.js

```diff
@@ -87,19 +80,10 @@ class ThreadActivityGraph extends React.PureComponent<Props, State> {
     });
   };
 
-  _takeCanvasRef = (canvas: HTMLCanvasElement | null) => {
-    this._canvas = canvas;
+  _setFillsQuerier = (fillsQuerier: ActivityFillGraphQuerier) => {
+    this._fillsQuerier = fillsQuerier;
   };
```

Comment:
This method sets the value of _fillsQuerier from the extracted canvas through a prop